pycloudlib/instance.py

```diff
@@ -99,7 +109,7 @@ def _wait_for_instance_start(self):
     def wait(self):
         """Wait for instance to be up and cloud-init to be complete."""
```

Comment:
I may be wrong here, but wait() now also always waits for the boot_id to change, via `_wait_for_boot_id()`. This means that if one calls `wait()` without also doing a reboot/restart of the instance, the `wait()` call will timeout with an error. Maybe we need a separate `wait_new_boot()` function, or `wait()` could grow a `require_new_boot_id` argument.